samples/bluetooth/hci_spi/README.rst

```diff
@@ -0,0 +1,19 @@
+Bluetooth: HCI SPI
+##################
+
+Overview
+********
+
+Expose Zephyr Bluetooth Controller support over SPI to another device/CPU using
+the Zephyr SPI HCI transport protocol (similar to BlueNRG).
+
+
+Requirements
+************
+
+* A board with SPI and BLE support
+
+Building and Running
+********************
+
+See :ref:`bluetooth setup section <bluetooth_setup>` for details.
```

Comment:
This references explains how to setup and run Bluetooth samples under QEMU and not on actual hardware.  Need some hardware specific instructions here.